R/calc-abundance.R

```diff
@@ -130,7 +130,7 @@ calc_frequency <- function(input, data_col, cluster_col = NULL, prefix = paste0
   res <- dplyr::mutate(
     res,
     .freq = dplyr::n_distinct(!!sym(cell_col)),
-    .pct  = (.data$.freq / .data$.n_cells) * 100
```

Comment:
We only need to remove .data for tidyselect expressions. There's actually only a few (~10 I think) instances of this that need to be corrected, mostly in `import_vdj()`